cucumber/authenticators_jwt/features/authn_jwt_token_schema.feature

```diff
@@ -0,0 +1,362 @@
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/ref: valid
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I successfully set authn-jwt "mandatory-claims" variable to value "<claims>"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00105E Failed to validate claim, claim name '<err>' is in denylist '["iss", "exp", "nbf", "iat", "jti", "aud"]'
+    """
+    Examples:
+    | claims        | err |
+    |   iss         |  iss   |
+    |   iat         |  iat   |
+    |   nbf         |  nbf   |
+    |   exp         |  exp   |
+    |   exp, iss    |  exp   |
+    |   exp, branch |  exp   |
+
+  Scenario Outline: ONYX-10857 - Standard claim in annotation - 401 Error
+    Given I have a "variable" resource called "test-variable"
+    And I successfully set authn-jwt "token-app-property" variable to value "host"
+    And I extend the policy with:
+    """
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/<claim>: valid
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00069E Role can't have one of these none permitted restrictions '["<claim>"]'>
+    """
+    Examples:
+    | claim   |
+    |   iss   |
+    |   iat   |
+    |   nbf   |
+    |   exp   |
+
+  Scenario: ONYX-10860 - Mandatory claims configured but not populated - 401 Error
+    Given I have a "variable" resource called "test-variable"
+    And I successfully set authn-jwt "token-app-property" variable to value "host"
+    And I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: mandatory-claims
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/sub: valid
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I issue a JWT token:
+    """
+    {
+      "ref":"valid",
+      "host":"myapp"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+     CONJ00037E Missing value for resource: cucumber:variable:conjur/authn-jwt/raw/mandatory-claims
+    """
+
+  Scenario: ONYX-10891 - Complex Case - Adding Mandatory Claim after host configuration
```

Comment:
Its not clear the full scenario you run here.
You run three `when` but by the scenario you run only two.
You run basic scenario(pass), add mandatory claim (fail) update the policy (pass) the last when is not clear for me

Reply:
Talked about it